coreneuron/sim/multicore.hpp

```diff
@@ -53,7 +53,7 @@ struct NrnFastImem {
     double* nrn_sav_d;
 };
 
-struct TrajectoryRequests {
+struct TrajectoryRequests: public MemoryManaged {
```

Comment:
unified memory fix. This is incomplete.

Reply:
I think I will just revert some of these changes and we can follow up separately. I think we are still quite far from having the full NEURON test suite passing with unified memory enabled.